### ISSUE TITLE
[Windows] [V1.2] vgmtrans.exe does not run/launch

### ISSUE THREAD

**Issue by thepyr0mann** [vgmtrans/vgmtrans#544]
### Environment details

* Operating System+version: Windows 10
* VGMTrans version: V1.2

### Describe the bug

Upon running vgmtrans.exe, nothing happens. GUI does not open, neither does any error message display whatsoever. 

### Expected behaviour

vgmtrans.exe runs.

### Additional context

The v1.1 version of vgmstrans.exe runs just fine on my PC, but the one before it did not.

**Comment by mikelow** [comment]
Not able to reproduce on Windows 10.

**Comment by mikelow** [comment]
Turns out I was able to reproduce this behavior on a system running an older version of Windows 10 (last update Dec 2023). Installing the latest Windows updates fixed the problem.

Could you please confirm if you have the latest updates installed and, if not, whether updating resolves the problem for you?

**Comment by Magirus-Deutz** [comment]
same thing on windows 11, if you using older version of windows 10 and 11 (older than update Dec 2023) then software doesn't launch

**Comment by quattj** [comment]
Can confirm that it is not loading on my version of Windows 10 64-bit, Core i7-10700 laptop.
My last "Quality Update" was installed 8/16/2024
Windows 10 22H2 build 19045.4780
https://support.microsoft.com/en-us/topic/august-13-2024-kb5041580-os-builds-19044-4780-and-19045-4780-2ef55b0d-bb01-41c8-8629-4146929792ad
Windows says I am up to date.

**Comment by possiblyamagicmage** [comment]
Same here on Windows 10, 64-bit. Intel i3-2120 CPU, version 1809 (LTSC).

**Comment by thepyr0mann** [comment]
> Turns out I was able to reproduce this behavior on a system running an older version of Windows 10 (last update Dec 2023). Installing the latest Windows updates fixed the problem.
> 
> Could you please confirm if you have the latest updates installed and, if not, whether updating resolves the problem for you?

Sorry for the late reply. I am absolutely sure I am up-to-date. My last one was just a few weeks ago, on the 11th of this month.

**Comment by Magirus-Deutz** [comment]
I moved my ssd with windows installed on the another computer and suddenly vgmtrans 1.2 is worked

The processor and graphics card gens are higher than the previous computer.

The previous my used computer's processor was AMD 6800H, right now i using AMD 7735HS

**Comment by Dr-Wily** [comment]
Same problem here. Tested on 4 Win10 PC, all up to date, nothing happen !

I think your compiler use a special CPU instruction that "older" CPUs don't have. Tested on gen 1 (Xeon), 3, 4 and 6 intel CPU. Annoying.

**Comment by mikelow** [comment]
> Same problem here. Tested on 5 Win10 PC, all up to date, nothing happen !
> 
> I think your compiler use a special CPU instruction that "older" CPUs don't have. Tested on gen 1 (Xeon), 3, 4, 6 and 8 intel CPU. Annoying.

Sorry to hear that. To clarify, you're saying the app wouldn't run (no window, no nothing) on 5 different generations of Xeon processors? What's the model of the most-recent gen CPU you tried?

**Comment by Dr-Wily** [comment]
Yeah, the app wouldn't run no window, no nothing, just sometime long cursor circle who disappear after 30s.

Only gen 1 CPU is a Xeon (E5620). The most recent is a Core i7 8750H.
